projects/dev.yaml

```diff
@@ -0,0 +1,85 @@
+# RESTBase project config for development environments
+paths:
+  /{api:v1}:
+    x-modules:
+      - spec:
+          info:
+            version: 1.0.0
+            title: Wikimedia REST API
+            description: Welcome to your RESTBase API.
+          x-route-filters:
+            - path: ./lib/normalize_title_filter.js
+              options:
+                redirect_cache_control: '{{options.purged_cache_control}}'
+          paths:
+            /page:
+              x-modules:
+                - path: v1/content.yaml
+                  options: '{{options}}'
+                - path: v1/content_segments.yaml
+                  options:
+                      purged_cache_control: '{{options.purged_cache_control}}'
+                      cx_host: '{{options.transform.cx_host}}'
```

Comment:
CXServer is not resource-hungry, but Apertium does sometimes consume some extra memory. However, given the scale for which the dev environment is intended that should not be a problem in practice. +1 on getting it into the dev environment.